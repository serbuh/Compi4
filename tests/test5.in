// Using lte/gte/neq operators

K x = 300 K;
F y = 150 F;
C z = 27 C; // Equals 300K

if(y>=x) {
  print "Pass 1";
}

if(y<=x) {
  print "This shouldn't be printed #1";
}

if(x>=z) {
  print "Pass 2";
}

if(x!=z) {
  print "This shouldn't be printed #2";
}

if(x<=z) {
  print "Pass 3";
}

if(x!=y) {
  print "Pass 4";
}
