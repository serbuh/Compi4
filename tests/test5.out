Pass 1
Pass 2
Pass 3
Pass 4
